### ISSUE TITLE
Please update Kramdown dependency to >=1.12.0

### ISSUE THREAD

**Issue by GarthDB** [github/pages-gem#380]
### Before submitting an issue, please be sure to

- [x] Read [the contributing instructions](https://github.com/github/pages-gem/blob/master/CONTRIBUTING.md#contributing-to-the-github-pages-gem)
- [x] Update to the latest Gem version (either `gem update github-pages` or `bundle update github-pages`)

### This issue affects

- [ ] The site generated by GitHub Pages
- [x] Building sites locally

### What did you do (e.g., steps to reproduce)

Added markdownlint (mdl) 0.4.0 to project. It has newly added support for ignoring yaml frontmatter.

### What did you expect to happen?

Be able to lint markdown

### What happened instead?

Couldn't install mdl because it depends on a higher version of kramdown (>= 1.12.0, ~> 1.12), but github-pages has 1.11.1

Here is the error message:

```sh
Bundler could not find compatible versions for gem "kramdown":
  In Gemfile:
    github-pages was resolved to 1, which depends on
      kramdown (= 1.0.2)

    mdl (= 0.4.0) was resolved to 0.4.0, which depends on
      kramdown (>= 1.12.0, ~> 1.12)
```

### Additional information

* Link to the live site (if applicable):
* Link to the source repo (if applicable):

**Comment by georgeperez** [comment]
See #334.

**Comment by GarthDB** [comment]
Closing, will be solved by #334 if anyone gets around to it.

**Comment by GarthDB** [comment]
Thanks @georgeperez
